WinImageBuilder.psm1

```diff
@@ -715,6 +743,7 @@ function Compress-Image {
         [Parameter(Mandatory=$true)]
         [string]$ImagePath
     )
+    Write-Log "Compress Image: $ImagePath..."

```

Comment:
Started image compression at: $ImagePath...

Reply:
And don't forget the Image compressed successfully at line: https://github.com/cloudbase/windows-openstack-imaging-tools/blob/experimental/WinImageBuilder.psm1#L765 before that if